Packages/io.chainsafe.web3-unity/Runtime/GUI/GuiScreen.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
+
+namespace ChainSafe.Gaming.GUI
+{
+    public class GuiScreen : MonoBehaviour, IGuiScreen
+    {
+        [field:SerializeField] public GuiLayer Layer { get; set; }
+        
+        public Canvas Canvas; // set layer sort order to canvas sort order?
+        public GameObject Content;
+        public Animation Animation;
```

Comment:
I prefer editing animations in editor rather than with tweens. You can play and tweak animations easily, making iterations much faster, leading to better animations.